core-plugins/widgets/File-batchsource.json

```diff
@@ -106,20 +119,17 @@
       "name": "schema",
       "widget-type": "schema",
       "widget-attributes": {
-        "default-schema": {
```

Comment:
i felt that this is not needed because we have added multiple formats. but it is good to have a default schema. so will put this back